### PR TITLE
updating docs to drop using `--wipe` flag

### LINKED ISSUE #439: update docs to drop the `--wipe` option
we dropping the `--wipe` flag from `restate-server`  so we need to update the docs to reflect that change

### PR DESCRIPTION
**Pull request by muhamadazmy**
Fixes #439 

> Question: How to preview mdx files locally

### CODE REVIEW

File: docs/develop/local_dev.mdx [hunk 1]
```diff
@@ -117,10 +117,10 @@ import clsx from "clsx";
 <details>
     <summary> Wiping Restate </summary>
 
-    To start the Restate Server from a clean slate and wipe ongoing invocations and K/V state:
+    To start the Restate Server from a clean slate, stop the server then wipe ongoing invocations and K/V state:
 
     ```shell
-    restate-server --wipe all
+    rm -rf <BASE_DIR>/${HOST}
```
Comment (tillrohrmann) [inline docs/develop/local_dev.mdx]: Maybe replace `${HOST}` with `<NODE_NAME>`. By default `<NODE_NAME> == ${HOST}` but one can configure it to something else.

File: docs/develop/local_dev.mdx [hunk 2]
```diff
@@ -117,10 +117,10 @@ import clsx from "clsx";
 <details>
     <summary> Wiping Restate </summary>
 
-    To start the Restate Server from a clean slate and wipe ongoing invocations and K/V state:
+    To start the Restate Server from a clean slate, stop the server then wipe ongoing invocations and K/V state:
```
Comment (tillrohrmann) [inline docs/develop/local_dev.mdx:120]:
```suggestion
    To start the Restate Server from a clean slate, stop the server and then delete the data directory:
```

File: docs/develop/local_dev.mdx [hunk 3]
```diff
@@ -146,7 +146,7 @@ import clsx from "clsx";
 
     [Cancel](/operate/invocation#cancelling-invocations) a single invocation or a batch of invocations.
     Use `--kill` to [kill](/operate/invocation#killing-invocations) the invocation.
-    To remove all invocations, restart the server with `--wipe all`.
+    To remove all invocations, stop the server then do `rm -rf <BASE_DIR>/<NODE_NAME>`.
```
Comment (tillrohrmann) [inline docs/develop/local_dev.mdx:149]: Maybe we should state that this deletes all state/data of the Restate server.

**Review by tillrohrmann — COMMENTED** [review]
Thanks for updating the documentation @muhamadazmy. The changes look good to me.

For a bit of context for @gvdongen: We want to discourage the wipe command because it is a) dangerous, b) should hopefully no longer be needed with our backwards compatibility and c) wouldn't work in the distributed set up as it is implemented right now. Instead we ask people to delete the data directory manually if they really need to. The wipe command itself is still there but it is hidden now (not visible when running `restate-server --help`).

**Comment by gvdongen** [comment]
I think there is a difference in how you use Restate locally for development/learning, vs production usage. 
I think when developing locally, it's perfectly fine to use this a lot. 

When developing locally, I still use it in quite some circumstances:
- Running an example, then another example and wanting to start from a clean slate
- Having many in flight invocations, delayed invocations and timers that I want to stop immediately to start over 
- It's still the fastest way to kill all invocations and state, if I have messed things up. We still don't have a `kill all invocations` command.

But yes, maybe doing `rm -rf` is also fine.

**Comment by muhamadazmy** [comment]
@gvdongen The argument `--wipe` is still available and you can use it. It's just hidden for normal users, and is then removed from documentations. You still can perfectly do `restate-server --wipe all` .

**Review by tillrohrmann — APPROVED** [review]
LGTM. +1 for merging.

**Pull request merged**